test/test_fat.rb

```diff
@@ -366,4 +366,38 @@ def test_change_rpath
       delete_if_exists(actual)
     end
   end
+
+  def test_add_rpath
+    pass
+  end
+
+  def test_delete_rpath
+    groups = FAT_ARCH_PAIRS.map do |arch|
+      ["hello.bin", "hello_actual.bin"].map do |fn|
+        fixture(arch, fn)
+      end
+    end
+
+    groups.each do |filename, actual|
+      file = MachO::FatFile.new(filename)
+
+      refute_empty file.rpaths
+      orig_npaths = file.rpaths.size
+
+      file.delete_rpath(file.rpaths.first)
+      assert file.rpaths.size < orig_npaths
```

Comment:
This will give unhelpful output if the assertion is violated. Use `assert_equal` (for `==)` and `assert_operator` (for `<` and other binary operators) here and in all the following newly added test cases.